common/src/View/MapDocument.cpp

```diff
@@ -622,6 +644,32 @@ namespace TrenchBroom {
             }
             kdl::vec_clear_and_delete(nodesToDelete);
 
+            // Clean up persistent IDs of any groups being pasted
+            auto persistentGroupIds = kdl::create_set(allPersistentGroupIds(*m_world.get()));
```

Comment:
Use explicit type here, I can't tell from context what it is

Reply:
This is clearer: 3b31997b588444aa1685a48271131f97955ced50